book/src/json-output.md

```diff
@@ -0,0 +1,157 @@
+# JSON output
+
+> Structured logging for `probe-run`.
+
+As an alternative to it's human-focused output, `probe-run` offers structured json output. There are two use-cases:
```

Comment:
I think it should be JSON not json throughout. Except when it's an argument like `--json-mode`.